File: ios/src/LynxFastImageModule.m
#import "LynxFastImageModule.h"

@implementation LynxFastImageModule

+ (NSString *)name {
  return @"LynxFastImageModule";
}

+ (NSDictionary<NSString *, NSString *> *)methodLookup {
  return @{
    @"setValue" : NSStringFromSelector(@selector(setValue:value:)),
    @"getValue" : NSStringFromSelector(@selector(getValue:)),
    @"clear" : NSStringFromSelector(@selector(clear)),
  };
}

- (void)setValue:(NSString *)key value:(NSString *)value {
}

- (nullable NSString *)getValue:(NSString *)key {
  return nil;
}

- (void)clear {
}

@end

File: android/src/main/java/com/example/lynxfastimage/LynxFastImageModule.java
package com.example.lynxfastimage;

import androidx.annotation.Nullable;
import com.lynx.jsbridge.LynxMethod;
import com.lynx.jsbridge.LynxNativeModule;
import com.lynx.tasm.behavior.LynxContext;
import com.example.lynxfastimage.generated.LynxFastImageModuleSpec;

@LynxNativeModule(name = "LynxFastImageModule")
public class LynxFastImageModule extends LynxFastImageModuleSpec {
  public LynxFastImageModule(LynxContext context) {
    super(context);
  }

  @Override
  @LynxMethod
  public void setValue(String key, String value) {
  }

  @Override
  @LynxMethod
  @Nullable
  public String getValue(String key) {
    return null;
  }

  @Override
  @LynxMethod
  public void clear() {
  }
}
